sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -226,7 +226,8 @@ abstract class HiveComparisonTest
       testCaseName: String,
       sql: String,
       reset: Boolean = true,
-      tryWithoutResettingFirst: Boolean = false) {
+      tryWithoutResettingFirst: Boolean = false,
+      assumption: Boolean = true) {
```

Comment:
I think this is OK, but might be more natural to either: call the parameter "skip" or something, and assume(!skip), or else, make it a function `() => Boolean` that the test actually tests later? calling the boolean result the assumption is a little funny.